Sources/IBLinterKit/Rules/DuplicateIdRule.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  DuplicateIdRule.swift
+//  IBLinterKit
+//
+//  Created by Eric Marchand on 2019/10/29.
```

Comment:
Could you drop headers?

Reply:
I can but maybe a new PR must remove all other headers